app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/bruker/BrukerRepository.kt

```diff
@@ -542,19 +542,36 @@ class BrukerRepositoryImpl(
         return@coRecord rows.toMap()
     }
 
-    private suspend fun oppdaterModellEtterDelete(aggregateId: UUID, callback: (tx: Transaction) -> Unit = {}) {
+    private suspend fun oppdaterModellEtterDelete(
+        aggregateId: UUID,
+        grupperingsid: String?,
+        callback: (tx: Transaction) -> Unit = {}
+    ) {
         database.transaction({
             throw RuntimeException("Delete", it)
         }) {
-            executeUpdate(""" DELETE FROM notifikasjon WHERE id = ?;""") {
+            if (grupperingsid != null) {
+                executeUpdate("""
+                    delete from notifikasjon n where n.grupperingsid = ?
+                        and merkelapp = (
```

Comment:
kunne vurdert å sende inn `merklapp` fra eventet, så slipper vi subquery.

Reply:
Det finnes ikke merkelapp på disse eventene

Reply:
Vi diskuterte dette og bestemte oss for å innføre merkelapp på delete eventene. 
Det gjøres her:
- https://github.com/navikt/arbeidsgiver-notifikasjon-produsent-api/pull/578

Reply:
3361ae2042580cb1a4a644fe78b4bd7a93e6c4fa